bigtable_input.py

```diff
@@ -0,0 +1,91 @@
+"""Read Minigo game examples from a Bigtable.
+"""
+
+from google.cloud import bigtable
+from google.cloud.bigtable import row_filters
+import logging
+import os
+import struct
+import tensorflow as tf
+
+
+_project_name = os.environ['PROJECT']
+_instance_name = os.environ['CBT_INSTANCE']
+_table_name = os.environ['CBT_TABLE']
+
+_bt_table = bigtable.Client(
+    _project_name).instance(
+        _instance_name).table(
+            _table_name)
+
+_tf_table = tf.contrib.cloud.BigtableClient(
+    _project_name,
+    _instance_name).table(
+        _table_name)
+
+
+_features = {
+    'x': tf.FixedLenFeature((), tf.string, ''),
+    'pi': tf.FixedLenFeature((), tf.string, ''),
+    'outcome': tf.VarLenFeature(dtype=tf.float32)
+    }
+
+
+def get_latest_game_number():
+  """Return the number of the next game to be stored.
+  """
```

Comment:
nit: can all fit on one line

Reply:
Will do. (Or will expand the comment.)

Reply:
Will do. (Or will expand the comment.)

Reply:
Thanks. Will fix (or expand the docstring content).